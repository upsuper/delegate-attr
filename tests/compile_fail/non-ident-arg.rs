use delegate_attr::delegate;

struct Foo(Vec<(u8, u8)>);

#[delegate(self.0)]
impl Foo {
    fn push(&mut self, (a, b): (u8, u8)) -> usize {}
}

fn main() {}
