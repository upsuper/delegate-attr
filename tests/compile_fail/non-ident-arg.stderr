error: expect an identifier
 --> $DIR/non-ident-arg.rs:7:24
  |
7 |     fn push(&mut self, (a, b): (u8, u8)) -> usize;
  |                        ^^^^^^
